C: Macro Argument warning in Arena

```
/home/runner/work/herb/herb/src/hb_arena.c:8:24: warning: macro argument should be enclosed in parentheses [bugprone-macro-parentheses]
    8 | #define KB(kb) (1024 * kb)
      |                        ^ 
      |                        ( )
```

--- a/src/include/macros.h
+++ b/src/include/macros.h
@@ -5,7 +5,7 @@
 
 #define MIN(a, b) ((a) < (b) ? (a) : (b))
 
-#define KB(kb) (1024 * kb)
+#define KB(kb) (1024 * (kb))
 
 #define MB(mb) (1024 * KB(mb))
 

